tests/e2e/e2e_test.go

```diff
@@ -211,3 +212,33 @@ func (s *IntegrationTestSuite) TestStateSync() {
 	err = chain.RemoveNode(stateSynchingNode.Name)
 	s.NoError(err)
 }
+
+func (s *IntegrationTestSuite) TestExpeditedProposals() {
```

Comment:
Is it possible to set the timeout specifically for this test case? Just in case it gets blocked due to multi-goroutine handling here

Reply:
There is a one minute timeout within the function the go routine is calling. Should I have a second timeout here as well? Would that be redundant or is it a good practice?

Reply:
Replied here: https://github.com/osmosis-labs/osmosis/pull/2341/files#r944758494